### PR TITLE
feat: add `get` support for bare-metal

### PR DESCRIPTION
**Pull request by shawnh2**
effects:
```bash
~ ./bin/gtctl cluster get test --bare-metal

+-----------+-----------+
| COMPONENT |    PID    |
+-----------+-----------+
| frontend  | .0: 82062 |
+-----------+-----------+
| datanode  | .0: 82059 |
+           +-----------+
|           | .1: 82060 |
+           +-----------+
|           | .2: 82061 |
+-----------+-----------+
| metasrv   |     82058 |
+-----------+-----------+
| etcd      |     82054 |
+-----------+-----------+
CREATION-DATE: 2023-08-23 23:11:57.432645 +0800 CST
GREPTIMEDB-VERSION: latest
ETCD-VERSION: v3.5.7
CLUSTER-DIR: /.gtctl/test
CLUSTER-CONFIG:
cluster:
    artifact:
        local: ""
        version: latest
    frontend:
        replicas: 1
        grpcAddr: ""
        httpAddr: ""
        postgresAddr: ""
        metaAddr: ""
        logLevel: ""
    meta:
        storeAddr: 127.0.0.1:2379
        serverAddr: 0.0.0.0:3002
        bindAddr: ""
        httpAddr: 0.0.0.0:14001
        logLevel: ""
    datanode:
        replicas: 3
        nodeID: 0
        rpcAddr: 0.0.0.0:14100
        httpAddr: 0.0.0.0:14300
        dataDir: ""
        walDir: ""
        procedureDir: ""
        logLevel: ""
etcd:
    artifact:
        local: ""
        version: v3.5.7
```

after the implementation of `get`, the `list`, `connect`, `update` cmd can be implemented.

related to #96

### CODE REVIEW

File: pkg/cmd/gtctl/cluster/get/test_data/pids/a/pid [hunk 1]
```diff
@@ -0,0 +1 @@
+123
```
Comment (zyy17) [inline pkg/cmd/gtctl/cluster/get/test_data/pids/a/pid:1]: nitpick: it's better to add the newline.

**Review by zyy17 — APPROVED** [review]
Seems great!

File: pkg/cmd/gtctl/cluster/get/test_data/pids/a/pid (hunk 1 above)
Comment (shawnh2) [reply pkg/cmd/gtctl/cluster/get/test_data/pids/a/pid:1]: sure

Comment (shawnh2) [reply pkg/cmd/gtctl/cluster/get/test_data/pids/a/pid:1]: newline seems unnecessary, the acutal pid file doesnt contain newline either

**Pull request merged**